gRPC/src/main/proto/cardData.proto

```
@@ -2,64 +2,72 @@ syntax = "proto3";
 
 package com.sub.grpc;
 
-// 온라인/오프라인/BOTH 구분
+import "google/protobuf/empty.proto";
+import "cardCompany.proto";
```

Buf (1.58.0)

import "cardCompany.proto": file does not exist

(COMPILE)